server/monitor/src/main/java/org/apache/accumulo/monitor/util/logging/AccumuloMonitorAppender.java

```diff
@@ -122,8 +122,8 @@ public void append(final LogEvent event) {
 
         var req = HttpRequest.newBuilder(uri).POST(BodyPublishers.ofString(jsonEvent, UTF_8))
             .setHeader("Content-Type", "application/json").build();
-        @SuppressWarnings("unused")
-        var future = httpClient.sendAsync(req, BodyHandlers.discarding());
+
+        httpClient.send(req, BodyHandlers.discarding());
```

Comment:
Changing this could be good to prevent using an unbounded number of threads for sending logs to the monitor. However, changing it would cause issues for any existing users if they did not wrap the monitor config with the [AsyncAppender](https://logging.apache.org/log4j/2.x/manual/appenders/delegating.html#AsyncAppender) with a `blocking=false` configuration.